### PR TITLE
v10 platform rename (#360)

### PR DESCRIPTION
**Pull request by bogdanudrescu**
This change is [Reviewable](https://reviewable.io/reviews/vaadin/flow-and-components-documentation/426)

### CODE REVIEW

**Comment by bogdanudrescu** [comment]
This is for master.

File: documentation/migration/7-tools-integrations.asciidoc [hunk 1]
```diff
@@ -8,61 +8,58 @@ layout: page
...
+The Bakery App Starter for Flow and Spring shows an example of this.
 
-== No CDI Support
+== CDI Support
 
-Vaadin 8 has a CDI Add-on enabling JEE features. This is currently missing from Vaadin 10.
-
-We have plans for adding it, but no timeline for this at the moment. To ensure it gets a priority over several other new features and enhancements,
-you should voice your concern in our https://vaadin.com/forum[Forum] or https://github.com/vaadin/flow[GitHub].
+There is a https://github.com/vaadin/cdi[CDI Add-on] enabling JEE features. This is currently part of Vaadin platform starting with V12 and also compatible with V10.
```
Comment (mstahv) [inline documentation/migration/7-tools-integrations.asciidoc]:
```suggestion
There is a https://github.com/vaadin/cdi[CDI Add-on] for easier CDI integration and to help using other Java EE features. This is currently part of Vaadin platform starting with V12 and also compatible with V10.
```

**Comment by bogdanudrescu** [comment]
___
*[pom.xml, line 54 at r6](https://reviewable.io/reviews/vaadin/flow-and-components-documentation/426#-LXOQqk9AK_2VAZ15OFH:-LXORPHQ0Je2j92X86fr:bwhwi9o) ([raw file](https://github.com/vaadin/flow-and-components-documentation/blob/94b73b41ed8efb98d2eb472a38852993dacc12c8/pom.xml#L54)):*
<details><summary><i>Previously, denis-anisimov (Denis) wrote…</i></summary><blockquote>

> ```
>         <flow.version>1.3.0.alpha3</flow.version>
>         <vaadin.version>13.0.0.alpha3</vaadin.version>
> ```

Why do you use release version instead of SNAPSHOT?
It looks very suspiciously.
</blockquote></details>

The build fails on current snapshot. We can remote this and wait until it's fixed.

**Comment by bogdanudrescu** [comment]
___
*[pom.xml, line 54 at r6](https://reviewable.io/reviews/vaadin/flow-and-components-documentation/426#-LXOQqk9AK_2VAZ15OFH:-LXOULujBPXmkcB4s1wW:b3kemzq) ([raw file](https://github.com/vaadin/flow-and-components-documentation/blob/94b73b41ed8efb98d2eb472a38852993dacc12c8/pom.xml#L54)):*
<details><summary><i>Previously, bogdanudrescu (Bogdan Udrescu) wrote…</i></summary><blockquote>

The build fails on current snapshot. We can remote this and wait until it's fixed.
</blockquote></details>

OK, let's wait. I'll put it back once the snapshot is fine. Thanks!

**Pull request merged**